ci/README.md

```diff
@@ -6,6 +6,20 @@ Many of these scripts contain more detailed documentation and options in comment
 
 Any file and directory added into this tree should be documented here.
 
+## Publishing a release
+
+1. Change the version of code-server in `package.json` and push this commit.
+1. CI will run and generate an NPM package and release packages that you can download
+   as artifacts on Github Actions.
+1. Create a new draft release with the built release packages.
+1. Run some basic sanity tests on one of the released packages.
+1. Publish.
```

Comment:
I think we should mention to include the VS Code version in the release notes, see https://github.com/cdr/code-server/issues/1501#issuecomment-611031429. Including a changelog is of course obvious but maybe we should mention that as well (which at some point we should probably start generating automatically).

Reply:
You got it.

I'd be in favour of a non autogenerated changelog, commits can get verbose. I'd rather we summarize the big changes ourselves.

Reply:
sgtm